tests/test_php_semantic.py

```
@@ -0,0 +1,383 @@
+        assert var_count >= 3, f"processData() should have >= 3 vars, got {var_count}"
+
+
+class TestSemanticPHPExtraction:
+    """Test that PHP function extraction works correctly."""
+
+    def test_php_extract_units_for_extraction(self, temp_php_project):
+        """_process_file_for_extraction should extract PHP functions with metadata."""
+        from tldr.semantic import _process_file_for_extraction
+        from tldr.cross_file_calls import build_project_call_graph
+
+        file_path = Path(temp_php_project) / "functions.php"
```

Ruff (0.14.13)

Local variable `file_path` is assigned to but never used

Remove assignment to unused variable `file_path`

(F841)